MAINTAINERS.yml

```diff
@@ -5830,3 +5830,22 @@ RX arch:
     - "area: RX"
   tests:
     - arch.rx
+
+Raspberry Pi 4 Model B Platforms:
+  status: maintained
+  maintainers:
+    - lenghonglin
+  collaborators:
+    - xingrz
+  files:
+    - boards/raspberrypi/rpi_4b/
+    - dts/arm64/broadcom/bcm2711.dtsi
+    - dts/bindings/serial/brcm,bcm2711-aux-uart.yaml
+    - dts/bindings/gpio/brcm,bcm2711-gpio.yaml
+    - drivers/serial/uart_bcm2711.c
+    - drivers/serial/Kconfig.bcm2711
+    - drivers/gpio/gpio_bcm2711.c
+    - drivers/gpio/Kconfig.bcm2711
```

Comment:
These can be simplified with a glob pattern, which will also future proof if new drivers are added for this platform.

Reply:
Thanks , how about new one ?